api/errors/errors.go

```diff
@@ -141,6 +141,7 @@ const (
 	ErrSSEEncryptedObject
 	ErrInvalidEncryptionParameters
 	ErrInvalidSSECustomerAlgorithm
+	ErrInvalidEncryptionAlgorithm
```

Comment:
Aren't `ErrInvalidSSECustomerAlgorithm` and `ErrInvalidEncryptionAlgorithm` the same?

Reply:
Almost. The new one is actual aws error https://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html. I'll drop the old one.